src/connection/server_support.c

```diff
@@ -2645,6 +2645,13 @@ css_process_server_server_connect (SOCKET master_fd)
   switch (STATIC_CAST (cubcomm::server_server, request))
     {
     case cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER:
+      // *INDENT-OFF*
+      if (!chn.send_int (static_cast < int >(cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER)))
+	{
+	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_NET_TRANSACTIONSERVER_CONNECTION, 1, chn.get_channel_id ());
```

Comment:
Printing channel ID does not help inform the user. The one information that could be relevant is the transaction server hostname, but for now it is not known. The server_channel connection protocol may have to be modified to include the hostname of the connecter.

I see two possible options here:
  - Do not include "at %s" part in the error message and do not print get_channel_id().
  - Let "at %s" part in the error message, print "unknown" for now and leave a todo to be replaced by the hostname of the connecter host.

Reply:
I'll leave out the %s for now.